### ISSUE TITLE
Opposing Force + Blue shift support

### ISSUE THREAD

**Issue by ghost** [eukara/freehl#10]
Once the Half-Life original game is playable could support be added for opposing force and blue shift especially for the data discs versions?

**Comment by eukara** [comment]
Opposing Force is located here: https://github.com/eukara/freegearbox

**Comment by perondeidad** [comment]
Did the 25th Anniversary Update break any compatibility?

**Comment by perondeidad** [comment]
> > Opposing Force is located here: https://github.com/eukara/freegearbox
> 
> Thanks! What about the Blue Shift addon? Also is the Gunman Chronicles still being worked on as well?

I guess so, the last update of the repository was 3 days ago

**Comment by eukara** [comment]
Yeah, slowly but surely completing more and more. Life keeps me busy but I won't stop doing this.
